src/consensus/pbft/libbyz/pre_prepare.cpp

```diff
@@ -474,4 +474,24 @@ const std::array<uint8_t, MERKLE_ROOT_SIZE>& Pre_prepare::
 int64_t Pre_prepare::get_ctx() const
 {
   return rep().ctx;
+}
+
+std::vector<int> Pre_prepare::get_prev_pp_valid_principals()
+{
+  if (rep().num_prev_pp_sig == 0)
+  {
+    return {};
+  }
+  std::vector<int> principals_with_proof;
+  // signatures start at
+  uint8_t* sigs = (uint8_t*)contents() + sizeof(Pre_prepare_rep) +
+    rep().rset_size + rep().n_big_reqs * sizeof(Digest);
+  auto s = sigs;
```

Comment:
Why create s? sigs doesn't seem to be otherwise reused?

Reply:
That's just leftover from something else, thanks for spotting it